wger/config/tests/test_gym_config.py

```diff
@@ -90,4 +90,4 @@ def test_update_userprofile(self):
         self.assertEqual(UserProfile.objects.filter(gym=gym).count(), 24)
 
         # 13 non-managers
-        self.assertEqual(GymUserConfig.objects.filter(gym=gym).count(), 13)
+        self.assertEqual(GymUserConfig.objects.filter(gym=gym).count(), 18)
```

Comment:
The Count should match the Comment above. `18`

Reply:
Okay. I have fixed that.